extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/music/GoogleMusicImporter.java

```diff
@@ -230,7 +230,8 @@ private String summarizeNewPlaylistItemResultErrors(
         // TODO(critical WIP-feature step): Add permanent failures.
         throw new CopyException("Permanent Failure:", e);
       } else if (StringUtils.contains(e.getMessage(), "Fail to find track matching")
-          || StringUtils.contains(e.getMessage(), "Missing ISRC in playlist item")) {
+          || StringUtils.contains(e.getMessage(), "Missing ISRC in playlist item")
+          || StringUtils.contains(e.getMessage(), "Max videos exceeded")) {
         // Skippable Failure: we skip this batch and log some data to understand it better
         // TODO(critical WIP-feature step): Add skippable failures.
         monitor.info(() -> "Skippable Failure:", e);
```

Comment:
nit: is `warn()` better here?

Reply:
Monitor only supports info, debug and server(error).

Reply:
your call; feel free to use severe - seems appropriate for failing to transfer something.